android/src/com/mapswithme/maps/widget/placepage/PlacePageView.java

```diff
@@ -1179,9 +1231,9 @@ public boolean onLongClick(View v)
       case R.id.ll__place_email:
         items.add(mTvEmail.getText().toString());
         break;
-      case R.id.ll__place_phone:
-        items.add(mTvPhone.getText().toString());
-        break;
+      //case R.id.ll__place_phone:
+      //  items.add(mTvPhone.getText().toString());
+      //  break;
```

Comment:
Possibly, should not comment, but just delete. It will stay in git history. Please, check in other places.